ui/src/trace_processor/http_rpc_engine.ts

```diff
@@ -32,6 +32,8 @@ export class HttpRpcEngine extends EngineBase {
   private websocket?: WebSocket;
   private connected = false;
   private disposed = false;
+  private queue: Blob[] = [];
+  private isProcessing = false;
```

Comment:
can you call this isProcessingQueue (or isQueueBusy or anything with Queue) so it's clear this is tied to the `queue`?